tests/cases/fourslash/codeFixImportNonExportedMember_all.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: /a.ts
+////declare function foo(): any;
+////declare function bar(): any;
+////declare function zoo(): any;
+////export { zoo }
+
+// @Filename: /b.ts
+////import { foo, bar } from "./a";
+
+goTo.file("/b.ts");
+verify.codeFixAll({
+    fixId: "importNonExportedMember",
+    fixAllDescription: ts.Diagnostics.Export_all_non_exported_member.message,
+    newFileContent: {
+        '/a.ts': `export declare function foo(): any;
+export declare function bar(): any;
+declare function zoo(): any;
+export { zoo }`

```

Comment:
I think the diff should actually be `export { foo, bar, zoo }` because no other declaration has an `export` modifier in this case - what do others think?

Reply:
I think your idea is more elegant, no matter how many export modifier exists, if there has an `export { }`, other unexported member should just added to `{}` in order. Is this understanding correct?

Reply:
@DanielRosenwasser when applyiing `getAllCodeActions ` strategy, two single text change `{ zoo, foo }` and `{ zoo, bar }` occurs at the same line, which emit a `Changes overlap` Failure, is there any way to solve this issue ?

Reply:
You'd have to write a custom getAllCodeActions handler, which would be difficult for me, at least. I'd just stick with your current code for an initial version and we can improve it later.